### ISSUE TITLE
[Feature Request] Show more Pokestop Details

### ISSUE THREAD

**Issue by HELLspawnTIM** [PokeAlert/PokeAlert#412]
I would like to ask if it's possible to add more Details to found Pokestops.  
- Name of Pokestop
- If Lure is ON, Trainername of the Trainer who activated the Lure Module

**Comment by PokeAlert** [comment]
if lure is on the poke stop is pink

**Comment by HELLspawnTIM** [comment]
Yes, i saw that. But could you also include the Trainername who placed the lure? 
And also include the Pokestop Name itself?

**Comment by edwardwong80** [comment]
In a Japanese PoGo map, I saw that a pokestop can show how much time left after the launch of lure module, can we have this info as well? (Sometimes we planned to have event and purchased lure module to release, but we want to prepare well with this info)

**Comment by PokeAlert** [comment]
@HELLspawnTIM Honest question: Why is the trainer so important to show?

**Comment by HELLspawnTIM** [comment]
@PokeAlert
Simply for the same reason it is important to show the trainer name when scanning a Gym:
To get in touch and in contact with other trainers instead od playing with just anonymous other players. Also to get known which trainers are interested in getting a certain area lured. 
Anyway, i find my first part of this feature request more Important:

Please include the name of the Pokestop found, in order to help us getting the correct Pokestop lured....
